core/src/test/java/nl/nn/adapterframework/receivers/SlowListenerWithPollGuard.java

```diff
@@ -25,21 +27,26 @@ public class SlowListenerWithPollGuard extends SlowPushingListener implements IP
     private Timer pollGuardTimer = null;
     private @Getter @Setter int mockLastPollDelayMs = 10;
     private @Setter Receiver<Message> receiver;
-    @Override
+	private SpringJmsConnector mockConnector;
+
+	@Override
+	public void configure() throws ConfigurationException {
+		DefaultMessageListenerContainer mockContainer = mock(DefaultMessageListenerContainer.class);
+		mockConnector = mock(SpringJmsConnector.class);
+		when(mockConnector.getLastPollFinishedTime()).thenAnswer(invocationOnMock -> System.currentTimeMillis() - mockLastPollDelayMs);
+		when(mockConnector.getThreadsProcessing()).thenReturn(new Counter(0));
+		when(mockConnector.getReceiver()).thenReturn(receiver);
+		when(mockConnector.getListener()).thenReturn(this);
+		when(mockConnector.getJmsContainer()).thenReturn(mockContainer);
+		when(mockConnector.getLogPrefix()).thenReturn("MockJmsConnector ");
+	}
+
+	@Override
```

Comment:
These changes should fix the 2 tests that caused trouble after updating Mockito.

1. The mocks were created in a different thread
2. The mocks were created while opening the Receiver, which was done under a TimeoutGuard.
3. After updating Mockito it took forever to create the mocks from this thread. Probably a Mockito bug.
4. This triggered the TimeoutGuard and thus it failed the test.

Now the timeout guard should not be triggered anymore because `configure()` doesn't run under the TimeoutGuard, and the mocks are now created in the main JUnit thread again so this should not trigger the Mockito issue.